packages/grafana-ui/src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { FunctionComponent } from 'react';
+import { Range, createSliderWithTooltip } from 'rc-slider';
+import { css } from 'emotion';
+import { stylesFactory, withTheme } from '../../themes';
+import { GrafanaTheme } from '@grafana/data';
+import { Themeable } from '../../types';
+
+export interface Props extends Themeable {
+  min: number;
+  max: number;
+  orientation: 'horizontal' | 'vertical';
+  reverse?: boolean;
+  size?: string;
```

Comment:
what does this do? what are the possible values?

Reply:
I will add the comment to make it more clear and change it to number to be even more clear. But basically, it sets width of horizontal slider and height of vertical slider. So you can change the size based on your usecase. 🙂

Reply:
The default value is 200px.

Reply:
I have changes the naming to lengthOfSlider, so it is more clear. Also, I have added possibility to customize lengthOfSlider in storybook.
![image](https://user-images.githubusercontent.com/30407135/74764792-ad9a0600-5282-11ea-8664-45746bee4895.png)